src/braket/aws/aws_quantum_job.py

```diff
@@ -528,10 +528,17 @@ def __eq__(self, other) -> bool:
     def __hash__(self) -> int:
         return hash(self.arn)
 
+    @staticmethod
+    def _get_device_region(device_arn: str) -> str:
+        try:
+            return device_arn.split(":")[3]
+        except IndexError:
+            raise ValueError(f"Device arn is not a valid format: {device_arn}")
```

Comment:
Would it make sense to include a global and regional arn as examples in the error output?
```suggestion
            raise ValueError(f"Device arn is not a valid format: {device_arn}"). Example of correct arns are:
            arn:aws:braket:::device/qpu/d-wave/Advantage_system1 or arn:aws:braket:us-west-2::device/qpu/d-wave/Advantage_system1
```

Reply:
Sure, that could make sense. Adding a region to a non-regionalized ARN will not work though, so we'll have to use M1 as an example. Also need to update the test.